### PR TITLE
Making Portfolio more prominent

### PR DESCRIPTION
**Pull request by sergei-shabanau**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -100,15 +100,89 @@ <h2 class="post-title">{{ post.title }}</h2>
...
+ </section> -->
  <!-- Posts end-->
 
+
+<section class="module">
+    <div class="container">
+        <div class="row">
+            <div class="col-md-6 m-auto text-center">
+                <h1>Our projects</h1>
+                <p class="lead">Some of our projects we have built with our favorite language Scala and functional programming.</p>
```
Comment (soujiro32167) [inline index.html:118]:
```suggestion
                <p class="lead">Some of our projects we have built with our favorite language Scala, using the functional programming paradigm.</p>
```

File: index.html [hunk 2]
```diff
@@ -100,15 +100,89 @@ <h2 class="post-title">{{ post.title }}</h2>
...
+            <div class="col-md-12">
+                <div class="space" data-MY="100px"></div>
+            </div>
+        </div>
+
+        <div class="row align-items-center" style="padding: 40px;border: 1px solid #f0f0f0;border-radius: .1875rem;">
+            <div class="col-lg-5"><img src="assets/images/services/1.jpg" alt=""></div>
+            <div class="col-lg-6 offset-lg-1">
+                <h3>Empowering domain experts to test and execute business logic from specifications</h3>
+                <p class="lead">With the addition of business oriented language we eliminated costly development cycles and reduced time-to-production of new features from weeks to hours.</p>
```
Comment (soujiro32167) [inline index.html:132]:
```suggestion
                <p class="lead">With the addition of business oriented language, we eliminated costly development cycles and reduced time-to-production of new features from weeks to hours.</p>
```

File: index.html [hunk 3]
```diff
@@ -100,15 +100,89 @@ <h2 class="post-title">{{ post.title }}</h2>
...
+
+        <div class="row">
+            <div class="col-md-12">
+                <div class="space" data-MY="120px"></div>
+            </div>
+        </div>
+
+        <div class="row align-items-center" style="padding: 40px;border: 1px solid #f0f0f0;border-radius: .1875rem;">
+            <div class="col-lg-6">
+                <h3>Reducing troubleshooting from days to hours for major mobile carrier</h3>
```
Comment (soujiro32167) [inline index.html:145]:
```suggestion
                <h3>Reducing troubleshooting from days to hours for a major mobile carrier</h3>
```

File: index.html [hunk 4]
```diff
@@ -100,15 +100,89 @@ <h2 class="post-title">{{ post.title }}</h2>
...
+                <p class="lead">By building customizable data pipelines and leveraging machine learning models we were able to reduce the troubleshooting and analysis of the failed network equipment tests from days to hours for the largest telecom industry in North America.</p>
```
Comment (soujiro32167) [inline index.html:146]:
```suggestion
                <p class="lead">By building customizable data pipelines and leveraging machine learning models, we were able to reduce the troubleshooting and analysis of the failed network equipment tests from days to hours for one of the largest telecom providers in North America.</p>
```

File: services.html [hunk 5]
```diff
@@ -158,115 +158,116 @@ <h1>Portfolio</h1>
...
+               <h3>Empowering domain experts to test and execute business logic from specifications</h3>
+               <br/>
+               <h5>With the addition of business oriented language we eliminated costly development cycles and reduced
+                   time-to-production of new features from weeks to hours.</h5>
+               <br/>
+               <p>
+                   We had to solve a problem of long development cycle. Technical specifications for extracting data from equipment status reports were created
+                   by domain experts and had to go through a full development process before being available for validation and deployment.
+               </p><p>
+               The amount of new features and changes was significant enough to occupy a team of developers and prevented the project to scale.
```
Comment (soujiro32167) [inline services.html:175]:
```suggestion
               The amount of new features and changes was significant enough to occupy a team of developers, preventing the project from scaling.
```

File: services.html [hunk 6]
```diff
@@ -158,115 +158,116 @@ <h1>Portfolio</h1>
...
-            The amount of new features and changes was significant enough to occupy a team of developers and prevented the project to scale.
-            </p><p>
-            We have introduced a custom programming language to describe data extractions with syntax closely followed the way specifications 
-            were originally written. Plus we've added necessary interpretation and execution capabilities to support internal language in the product. </p>
-            <p>
-            This allowed domain experts to express data extraction logic directly as programs, ready for testing and execution immediately, completely 
-            bypassing the long and expensive development cycle.
-            </p>                        
-           
+              <h3>Reducing troubleshooting from days to hours for major mobile carrier</h3>
```
Comment (soujiro32167) [inline services.html:193]:
```suggestion
              <h3>Reducing troubleshooting from days to hours for a major mobile carrier</h3>
```

**Comment by sergei-shabanau** [comment]
@soujiro32167 
Updated text according to your suggestions.

**Comment by soujiro32167** [comment]
@sergei-shabanau great stuff! I like how the new content is meant for the executive / manager type =]

**Pull request merged**